Source/Scene/ImplicitSubtree.js

```diff
@@ -64,6 +65,7 @@ export default function ImplicitSubtree(
   this._readyPromise = when.defer();
 
   // properties for 3DTILES_metadata
+  this._metadata = undefined;
   this._metadataTable = undefined;
   this._metadataExtension = undefined;
```

Comment:
Hm... while we're at it, could we rename `metadataTable -> tileMetadataTable` and `metadataExtension -> tileMetadataExtension`? This is more explicit, and will make things easier to understand in a future PR when you'll be updating tile metadata and adding content metadata.